Formula/blast.rb

```diff
@@ -17,6 +17,16 @@ class Blast < Formula
   conflicts_with "proj", :because => "both install a `libproj.a` library"
 
   def install
+    # Fix shebang lines and JSON dependency
+    inreplace "c++/src/app/blast/update_blastdb.pl", "#!/usr/bin/perl", "#!/usr/bin/env perl"
+    inreplace "c++/src/app/blast/update_blastdb.pl", "JSON", "JSON::PP"
+    inreplace "c++/src/app/blast/update_blastdb.pl", "from_json", "decode_json"
+
+    sed_script_file = Tempfile.new("sed_script.sed")
+    sed_script_file.puts("s/direct$/&:$PATH/")
+    sed_script_file.close
+    system "sed", "-i~", "-f", sed_script_file.path, "c++/src/app/blast/get_species_taxids.sh"
+
```

Comment:
Please remove this chunk.

Reply:
If I remove this, it breaks `get_species_taxids.sh` for anyone who already has edirect installed.

Reply:
What's the error message that you see without this chunk applied?
What's the complete contents of the line being modified by this `sed` script?
Please use `inreplace` rather than `sed`.

Reply:
> What's the error message that you see without this chunk applied?

The edirect tools aren't found.

> What's the complete contents of the line being modified by this `sed` script?

`export PATH=/bin:/usr/bin:/am/ncbiapdata/bin:$HOME/edirect`

> Please use `inreplace` rather than `sed`.

Done: https://github.com/Homebrew/homebrew-core/pull/47267/commits/735d3544a05b8c11c9884597220025b10efb6e8a